backend/api/views/simpleSignupRequestView.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from django.shortcuts import redirect
+from ..serializers import SignupSerializer
+
+@api_view(['POST'])
+def createSignupRequest(request):
```

Comment:
I didn't see permission declares here, I am not sure if you can make a post to this view. The login pr has made all views default to require login to make a post.

Reply:
This is not in the latest commit log. I added the permission to it